README.md

```diff
@@ -1,35 +1,65 @@
-# WIP  temp-alastriaID-truffle-contracts --- 
```

Comment:
Can we add section about how to use the scripts in the package and how to deploy contracts in a local env?